api/jobs.go

```diff
@@ -24,84 +29,125 @@ var (
 	serverErrorMessage = "internal server error"
 )
 
-// CreateJobHandler returns a function that generates a new Job resource containing default values in its fields.
-func (api *API) CreateJobHandler(ctx context.Context) http.HandlerFunc {
-	return func(w http.ResponseWriter, req *http.Request) {
-		ctx := req.Context()
+// CreateJobHandler generates a new Job resource and a new ElasticSearch index associated with it	.
+func (api *API) CreateJobHandler(w http.ResponseWriter, req *http.Request) {
```

Comment:
Correct me if I'm wrong, but this is no longer "creating" the job handler - it is the job handler, no?